lib/Zonemaster/Backend/Config.pm

```diff
@@ -58,22 +72,130 @@ sub BackendDBType {
     }
 }
 
-sub DB_user {
-    my ($self) = @_;
+=head2 MYSQL_database
+
+Returns the L<MYSQL.database|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#mysql>
+property from the loaded config, or the L<DB.database_name|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#db>
+property if it is unspecified.
+
+=cut
 
-    return $self->{cfg}->val( 'DB', 'user' );
+sub MYSQL_database {
+    my ( $self ) = @_;
+
+    return $self->{cfg}->val( 'MYSQL', 'database' ) // $self->{cfg}->val( 'DB', 'database_name' );
```

Comment:
Will it fail if `DB.database_name` is unspecified ?

Reply:
It'll return undef, and the caller will most likely fail at some point. But it's like that for all the properties. I plan to take care of this in #685.